README.md

```diff
@@ -53,7 +53,7 @@ or
         cd mist
         sbt -DsparkVersion=1.5.2 assembly 
     
-* Create [configuration file](#configuration)
+* Create [configuration file](https://github.com/Hydrospheredata/mist/blob/master/configs/docker.conf)
```

Comment:
@werowe could you explain this change please? Here we point to Mist config file. Docker config is a different one.